apidocs/src/check_permission.md

```diff
@@ -0,0 +1,50 @@
+## Check permission [/resources/{account}/{kind}/{identifier}?check=true&role={role}&privilege={privilege}]
+
+### Check permission [GET]
+
+Checks whether a role has a privilege on a resource. For example: is this Host
```

Comment:
In the rest of the docs, we write `For example,` with a comma instead of a colon. Let's make this consistent.